peachjam/models/judgment.py

```diff
@@ -72,10 +72,31 @@ def __str__(self):
         return self.name
 
 
+class CourtRegistry(models.Model):
+    court = models.ForeignKey(
+        Court,
+        on_delete=models.CASCADE,
+        null=True,
+        related_name="court_registry",
+    )
+    name = models.CharField(_("name"), max_length=1024, null=False, blank=False)
+
+    class Meta:
+        verbose_name = _("court registry")
+        verbose_name_plural = _("court registries")
+        unique_together = ("court", "name")
+
+    def __str__(self):
+        return f"{self.name} Registry - {self.court}"
```

Comment:
```suggestion
        return f"{self.name} - {self.court}"
```

Not all registries are called as such. eg. in Kenya they're court stations.